sdk/src/Core/Amazon.Runtime/Internal/InternalConfiguration.cs

```diff
@@ -130,17 +130,31 @@ private bool TryGetEnvironmentVariable(string environmentVariableName, out strin
                 return null;
             }
 
-            var converter = TypeDescriptor.GetConverter(typeof(T?));
```

Comment:
nitpick: You can remove `using System.ComponentModel;` now.

Reply:
Removed all of the unused namespaces.